.github/workflows/functional-test-cloud.yaml

```diff
@@ -348,10 +348,48 @@ jobs:
           append: true
           message: |
             :x: Test recipe publishing failed
+  
+  publish-test-bicep-types:
+  name: Publish Radius bicep types to ACR
+    runs-on: ubuntu-latest
+    steps: 
+      - name: Check out code
+        uses: actions/checkout@v4
+      - name: Parse release version and set environment variables
+        run: python ./.github/scripts/get_release_version.py
+      - name: Set up Go ${{ env.GOVER }}
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GOVER }}
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '18'
+      - name: Generate Bicep extensibility types from OpenAPI specs
+        run: |
+          make generate-bicep-types VERSION=${{ env.REL_VERSION == 'edge' && 'latest' || env.REL_VERSION }}
+      - name: Upload Radius Bicep types artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: radius-bicep-types
+          path: ./hack/bicep-types-radius/generated
```

Comment:
could you remind me what's we decided for this directory? Will it be removed or renamed to something less hacky sounding than `hack`? :)

Reply:
Ideally, we'd have this in a separate repository the same way we have `bicep-types-aws`. I can rename it for now if that'd be better

Reply:
I see, I think at least renaming will definitely help because questions around naming for this come up often. I'm fine with following up on this in a separate PR to keep this one scoped, feel free to log an issue to drive the discussion/follow up.